apps/emqx_bridge/src/emqx_action_info.erl

```diff
@@ -259,9 +247,10 @@ action_info_modules(App) ->
 initial_info_map() ->
     #{
         action_type_names => #{},
-        bridge_v1_type_to_action_type => #{},
-        action_type_to_bridge_v1_type => #{},
-        action_type_to_connector_type => #{},
+        % [FIXME] remove 'mqtt/http' once implemented
+        bridge_v1_type_to_action_type => #{mqtt => mqtt, http => http},
+        action_type_to_bridge_v1_type => #{mqtt => mqtt, http => http},
+        action_type_to_connector_type => #{mqtt => mqtt, http => http},
```

Comment:
Strange that http is needed to be added here, since it's already migrated from `webhook` to `http` :thinking: .

Reply:
I think this is a side-effect of renaming the http bridge in V1 schema: https://github.com/emqx/emqx/blob/79e7327042d2a6cf9010681088a811fb57f32512/apps/emqx_bridge/src/schema/emqx_bridge_schema.erl#L171-L175

I believe this renaming shouldn't have been done, and that we should treat v1 schema as immutable as possible now that we're migrating to actions/connectors/sources.